Source/WebCore/dom/Range.h

```diff
@@ -90,6 +90,7 @@ class Range final : public AbstractRange, public CanMakeSingleThreadWeakPtr<Rang
 
     Ref<DOMRectList> getClientRects() const;
     Ref<DOMRect> getBoundingClientRect() const;
+    static Ref<DOMRect> boundingClientRect(const SimpleRange&);
```

Comment:
We should change this to a non-member function at some point but it's fine like this.